translations/ui-tenant-settings/en.json

```diff
@@ -1,6 +1,21 @@
 {
   "meta.title": "Tenant",
 
+  "settings.title": "Tenant settings",
+  "settings.address.title": "Tenant settings - Addresses",
+  "settings.locale.title": "Tenant settings - Language and localization",
+  "settings.plugins.title": "Tenant settings - Preferred plugins",
+  "settings.sso.title": "Tenant settings - SSO settings",
+  "settings.service.title": "Tenant settings - Service points",
+  "settings.location.institutions.title": "Tenant settings - Institutions",
+  "settings.location.libraries.title": "Tenant settings - Libraries",
+  "settings.location.campuses.title": "Tenant settings - Campuses",
+  "settings.location.locations.title": "Tenant settings - Locations",
+  "settings.newService.title": "Tenant settings - Create new service point",
+  "settings.items.title": "Tenant settings - {item}",
+  "settings.location.creatNew.title": "Tenant settings - create new location",
```

Comment:
`<petty>`
1. As above, change the key to either `createNew` or just `create`. 
2. Capitalize values consistently across entries. Other values use "Tenant settings - Uppercase" so do that here, too.

`</petty>`

Reply:
Done!